index.html

```diff
@@ -103,65 +109,60 @@ <h2>Version Control comes to rescue</h2>
+                            <li class="fragment">Code revisions (commit) that you made would be available to only you locally unless you push to remote repo.
+                            </li>
+                            <li class="fragment">Create and support parallel branches, work locally, isolate your work and merge seamlessly to collaborate.</li>
+                            <li class="fragment">It takes snapshots of changes and treat files as stream of snapshots over time.
+                            </li>
+                        </ul>
                     </p>
                 </section>
                 <section>
                     <p>
-                    <ul>
-                        <li class="fragment">It compresses the .git contents</li>
-                        <li class="fragment">It has integrity</li>
-                        <li class="fragment">The mechanism that Git uses for this checksumming is called a SHA-1 hash. This
-                            is a 40-character string composed of hexadecimal characters (0–9 and a–f) and calculated based
-                            on the contents of a file or directory structure in Git. A SHA-1 hash looks something like this:
-                            <pre><code class="hljs" data-trim
-                                       contenteditable>24b9da6552252987aa493b52f8696cd6d3b00373</code></pre>
-                        </li>
-                    </ul>
+                        <ul>
+                            <li class="fragment">It compresses the .git contents</li>
+                            <li class="fragment">It has integrity</li>
+                            <li class="fragment">The mechanism that Git uses for this checksumming is called a SHA-1 hash. This is a 40-character string composed of hexadecimal characters (0–9 and a–f) and calculated based on the contents of a file or directory structure
+                                in Git. A SHA-1 hash looks something like this:
+                                <pre><code class="hljs" data-trim contenteditable>24b9da6552252987aa493b52f8696cd6d3b00373</code></pre>
+                            </li>
+                        </ul>
                     </p>
                 </section>
                 <section>
                     <p>
-                    <ul>
-                        <li class="fragment">You will see these hash values all over the place in Git because it uses them
-                            so much. In fact, Git stores everything in its database not by file name but by the hash value
-                            of its contents.
-                        </li>
-                    </ul>
+                        <ul>
+                            <li class="fragment">You will see these hash values all over the place in Git because it uses them so much. In fact, Git stores everything in its database not by file name but by the hash value of its contents.
```

Comment:
Trim this.

Reply:
Done. Converted.
Git stores everything in its database not by file name but by the hash value of its contents. These hash values are used very frequently.